packages/arco-lib/src/components/Link.tsx

```diff
@@ -12,8 +12,9 @@ const exampleProperties: Static<typeof LinkPropsSpec> = {
   disabled: false,
   hoverable: true,
   status: 'default',
-  href: 'https://www.smartx.com/',
+  href: '#',
   content: 'Link',
+  target: '_self',
```

Comment:
Maybe users don't know what does `_self` `_blank` mean. Can we translate `target` to boolean and rename it with a understandable name, like `OpenInNewPage`?

Reply:
👍

Reply:
done